Code.gs

```diff
@@ -26,7 +26,6 @@ var removeEventsFromCalendar = true;   // If you turn this to "true", any event
 var addAlerts = true;                  // Whether to add the ics/ical alerts as notifications on the Google Calendar events
 var addOrganizerToTitle = false;       // Whether to prefix the event name with the event organiser for further clarity 
 var descriptionAsTitles = false;       // Whether to use the ics/ical descriptions as titles (true) or to use the normal titles as titles (false)
-var defaultDuration = 60;              // Default duration (in minutes) in case the event is missing an end specification in the ICS/ICAL file
```

Comment:
There is no real need for a default Duration as Google Calendar accepts Events with 0min Duration (which is perfectly fine according RFC as well - see other comments).

Reply:
My original thought with `defaultDuration` was that the user could specify a duration in the case of "non-all-day events" where the duration and end datetime are not specified. Do you think we should just allow events to be 0 minutes in that case?

Reply:
I don't really see the need of defining a own default while the RFC already has one.
I would totally agree if ical did not allow to define a custom duration for events, but as of now i think we should aim for creating an as accurate representation of the input as possible, which includes 0 minute events.